### PR TITLE
Remove verbose JPA annotations from the quickstarts

### PR DESCRIPTION
**Pull request by svinther**
These annotation parameters are redundant and can be removed as discussed here https://github.com/quarkusio/quarkus/pull/11728

### CODE REVIEW

**Comment by gastaldi** [comment]
For some reason this PR is not targeted to the `development` branch. @svinther can you check?

**Comment by svinther** [comment]
@gastaldi should be better now

File: optaplanner-quickstart/src/main/java/org/acme/optaplanner/domain/Lesson.java [hunk 1]
```diff
@@ -20,7 +19,7 @@
 
     @PlanningId
     @Id
-    @GeneratedValue(strategy = GenerationType.AUTO)
+    @GeneratedValue()
```
Comment (gastaldi) [inline optaplanner-quickstart/src/main/java/org/acme/optaplanner/domain/Lesson.java:22]: I think we can remove the extra `()` for clarity

Comment (svinther) [reply optaplanner-quickstart/src/main/java/org/acme/optaplanner/domain/Lesson.java:22]: I absolutely agree, let me update the PR next morning

Comment (svinther) [reply optaplanner-quickstart/src/main/java/org/acme/optaplanner/domain/Lesson.java]: @gastaldi extra `()` are now removed

**Comment by gastaldi** [comment]
Awesome, thanks!

**Pull request merged**